srcpkgs/zsh-autosuggestions/INSTALL.msg

```diff
@@ -0,0 +1,3 @@
+Don't forget to source zsh-autosuggestions in your .zshrc file:
+
```

Comment:
This is not necessary

Reply:
@maxice8 Should I remove the message, or the whole *INSTALL.md*?